README.md

```diff
@@ -1,10 +1,44 @@
 # Stripes Vendor Dll
 
-Copyright (C) 2017-2020 The Open Library Foundation
+Copyright (C) 2017-2021 The Open Library Foundation
 
 This software is distributed under the terms of the Apache License,
 Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
+
+## Introduction
+
+Stripes Vendor Dll is a tool which can be used to create a [DLL bundle](https://webpack.js.org/plugins/dll-plugin/). The bundle currently includes all 3rd party dependencies (react, moment, etc) found in [stripes](https://github.com/folio-org/stripes/blob/master/package.json) plus couple additional stripes dependencies (stripes-components, stripes-connect,stripes-logger,stripes-util).
+
+## How to use it:
+
+There are two way to consume it:
+
+* Add `@folio/stripes-vendor-dll` to `package.json` in the ui module / platform you want to build:
```

Comment:
In `dependencies`?

Reply:
You are fast Mike :). I think if you are running without `NODE_ENV=production` `devDependencies` should work fine otherwise it should be in `dependencies`. Also I posted couple more details about the limitations with this current setup here: https://issues.folio.org/browse/FOLIO-2673